### PR TITLE
POWR-3439 cleanups from after Sprint 60 releases

### PR DESCRIPTION
**Pull request by joshuami**

### CODE REVIEW

File: web/private/scripts/slack_deploy_notification.php [hunk 1]
```diff
@@ -162,12 +162,11 @@ function _get_secrets($requiredKeys, $defaults)
 /**
  * Send a notification to slack
  */
-function _slack_notification($slack_url, $channel, $username, $text, /*$attachment,*/ $alwaysShowText = true)
+function _slack_notification($slack_url, $username, $text, /*$attachment,*/ $alwaysShowText = true)
 {
   // $attachment['fallback'] = $text;
   $post = array(
     'username' => $username,
-    'channel' => $channel,
```
Comment (RichardDavies) [inline web/private/scripts/slack_deploy_notification.php:170]: Why didn't you remove username here as well? If you're not going to pass in that parameter you'll need to remove it here too. Are you sure this function with work without the username and channel values?

**Review by RichardDavies — CHANGES_REQUESTED** [review]
I'm not sure your changes to the _slack_notification function parameters are all valid.

File: web/private/scripts/slack_deploy_notification.php [hunk 2]
```diff
@@ -162,12 +162,11 @@ function _get_secrets($requiredKeys, $defaults)
 /**
  * Send a notification to slack
  */
-function _slack_notification($slack_url, $channel, $username, $text, /*$attachment,*/ $alwaysShowText = true)
+function _slack_notification($slack_url, $username, $text, /*$attachment,*/ $alwaysShowText = true)
```
[the remark this answers is not in the document]
Comment (joshuami) [reply web/private/scripts/slack_deploy_notification.php:165]: It does pass the $username still. The $user name is a default variable near the start of the file.

File: web/private/scripts/slack_deploy_notification.php (hunk 1 above)
Comment (joshuami) [reply web/private/scripts/slack_deploy_notification.php:170]: Okay... trying again with a focus on only touching the channel.

**Review by RichardDavies — APPROVED** [review]
Ok, I think it looks good now.

**Pull request merged**